resources/js/Atoms/Textarea.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+
+
+interface iFieldProps {
+    placeholder?: string;
+}
+
+export const Textarea: React.FC<iFieldProps> = ({
+    placeholder
+}) => {
+    return (
+        <div>
+            <label className="block text-sm font-medium text-gray-700">
+                Textarea
+            </label>
+             <textarea
+             placeholder={placeholder}
+             onChange={
+                 (e: React.FormEvent<HTMLTextAreaElement>) =>
+                     console.log(e)
```

Comment:
you could change the console.log to console.log(e.currentTarget.value) to log out the typed value

Reply:
good call! changed it. Thank you Katja :D